app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java

```diff
@@ -26,23 +25,11 @@
 import mozilla.components.concept.storage.LoginsStorage;
 import mozilla.components.lib.fetch.httpurlconnection.HttpURLConnectionClient;
 
-import org.chromium.base.ApplicationStatus;
-import org.chromium.base.CommandLine;
-import org.chromium.base.ContextUtils;
-import org.chromium.base.PathUtils;
-import org.chromium.base.library_loader.LibraryLoader;
-import org.chromium.base.library_loader.LibraryProcessType;
-import org.chromium.content_public.browser.BrowserStartupController;
-import org.chromium.content_public.browser.DeviceUtils;
 import org.chromium.content_public.browser.WebContents;
-import org.chromium.ui.base.ResourceBundle;
 import org.chromium.wolvic.TabJni;
 
 public class RuntimeImpl implements WRuntime {
-    static String LOGTAG = SystemUtils.createLogtag(RuntimeImpl.class);
-
-    private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "content_shell";
-
+    private BrowserInitializer mBrowserInitiailizer;
```

Comment:
Nit: typo `mBrowserInitiailizer` -> `mBrowserInitializer`

Reply:
Oh, thanks for cathing it. I'll fix it.